File: typescript/base-hmi-types/src/screens/screen/HmiScreenMaster.ts
import { HmiScreenBase } from "../base/HmiScreenBase.js";
import { HmiScreenKind } from "../base/HmiScreenKind.js";
import { HmiObjectType } from "../base/HmiObjectType.js";

export class HmiScreenMaster extends HmiScreenBase {
  constructor() {
    super();
    this.hmiObjectType = HmiObjectType.HmiScreenMaster;
    this.kind = HmiScreenKind.Template;
  }
}

File: src/BaseHmiTypes/Screens/HmiScreenMaster.cs
using BaseHmiTypes.Screens.Base;

namespace BaseHmiTypes.Screens;

public class HmiScreenMaster : HmiScreenBase
{
    public HmiScreenMaster()
    {
        HmiObjectType = BaseHmiTypes.Screens.Base.HmiObjectType.HmiScreenMaster;
        Kind = HmiScreenKind.Template;
    }
}
